Gemfile.lock

```diff
@@ -499,38 +503,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
```

Comment:
### [pagy](https://github.com/ddnexus/pagy)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.6.0 → 3.14.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/pagy/3.6.0/3.14.0)_]





<details>
<summary>Commits</summary>

A change of **132** commits. See the full changes on [the compare page](https://github.com/ddnexus/pagy/compare/3.6.0...3.14.0).

These are the first 10 commits:
- (846eece) [fix in README.md](https://github.com/ddnexus/pagy/commit/846eece337362597ff93837faa852d9c1df50b7e)
- (77c045d) [add: tutorial link in ReadMe - complex search forms (#194)](https://github.com/ddnexus/pagy/commit/77c045df235d290c18649238ef9fdc06cf6754d8)
- (c290dca) [update incorrect tutorial link (#198)](https://github.com/ddnexus/pagy/commit/c290dcabf5dd844f1701f532f890f6d27ec4c8d0)
- (8b6f81f) [added pagy-cursor link in README](https://github.com/ddnexus/pagy/commit/8b6f81f639b1a80eb1ed3eebddb10311031c6b91)
- (ee1b1f0) [Update fr.yml (#195)](https://github.com/ddnexus/pagy/commit/ee1b1f0f3b9d1b14122c7fd654a9c8c2ad761f3c)
- (439676e) [Fixed previous translation for Spanish (#199)](https://github.com/ddnexus/pagy/commit/439676e06adcd17e78bdb5b362a004cc8426298b)
- (7775155) [replaced Pagy::Frontend::MARK with Pagy::PAGE_PLACEHOLDER (M…](https://github.com/ddnexus/pagy/commit/7775155fc73845e91ae157a5612e1fb46b7b3fe3)
- (3341dcc) [created ITEMS_PLACEHOLDER constant](https://github.com/ddnexus/pagy/commit/3341dcc6d29d0520b0456b1976860e3d3fff2d48)
- (224748a) [fix for resurrected-by-mistake line in README](https://github.com/ddnexus/pagy/commit/224748a20eec075e71dd2e5a4725209014dc5ff5)
- (b627e55) [add italian translation (#202)](https://github.com/ddnexus/pagy/commit/b627e55ace83651c026d234c6adf299b655ff7e7)


</details>